src/display/svg.js

```diff
@@ -689,14 +689,16 @@ SVGGraphics = (function SVGGraphicsClosure() {
           x += -glyph * fontSize * 0.001;
           continue;
         }
-        current.xcoords.push(current.x + x * textHScale);
+        var character = glyph.fontChar;
+        if (character !== ' ') {
```

Comment:
even if `character == ' '` it does not mean it's "empty", glyph can still display something.